package.json

```diff
@@ -17,6 +17,7 @@
     "refresh-workspace-hosts": "yarn workspace @prairielearn/prairielearn start --refresh-workspace-hosts-and-exit"
   },
   "dependencies": {
+    "clsx": "^2.1.1",
```

Comment:
Note that you should add this to `apps/prairielearn/package.json`, not this file in the root of the repository.